unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/InputController/InputController.cs

```diff
@@ -631,7 +632,16 @@ public static Boolean PassModifierKeys(KeyCode[] modifierKeys)
             }
             else
             {
-                if (modifierKeys.Contains(keyCode) != pressed)
+                // Very confusing function, but this should avoid allocations and LINQ
+                var contains = false;
```

Comment:
it was done in [this PR](https://github.com/decentraland/unity-renderer/pull/4186/files#diff-802617846bec437693e2fb4e126ce3ce8ff3777de96f79a31625d9b5a7ded0c8) as well (under review)